fms/models/hf/utils.py

```diff
@@ -130,3 +135,68 @@ def to_hf_api(model: nn.Module, **override_config_kwargs) -> "HFModelArchitectur
 
     hf_adapted_cls = _fms_to_hf_adapt_map[model_type]
     return hf_adapted_cls.from_fms_model(model, **override_config_kwargs)
+
+
+def get_model(
```

Comment:
it seems like we should name this something more specific to avoid conflicts with models.get_model. something like `get_hf_model_as_fms` (though that seems maybe a bit too long)

Reply:
ya i was thinking this as well. Maybe `get_model_from_hf`?

Reply:
or maybe `fms.models.hf.as_fms_model(hf_id_or_path, ...)` would be clear enough?